post.md

```diff
@@ -1,32 +1,30 @@
 # Robot Framework - using Request library for control RPI GPIO's
 
 Requests library is one of the most popular libraries implemented for Robot
-Framework. It is very important for testing scattered applications, so this is
-the first library I got to know in the Robot Framework. The library requires 
-installing `robotframework-requests` and `requests` for proper working. 
-`Collections` library is a standard built-in library.
+Framework. It is very important for testing distributed applications, so this is
+the first library I got to know in the Robot Framework. 
 
-For checking `Request` library we can install Pi-GPIO-Server on Raspberry Pi.
-Server is able to control state of input and output of GPIOS using Rest API.
+To test `Request` library we can install Pi-GPIO-Server on Raspberry Pi.
+Server is able to control state of input and output of GPIOs using REST API.
 
 Project is availiable on link: [Project weekend - GPIO Server](https://github.com/projectweekend/Pi-GPIO-Server)
 
-In the Rest Api we use methods:
+In the REST Api we used following methods:
 - Delete
 - Get
 - Options
 - Post
 - Put
 - Patch
 
-To control GPIO's on the RPI we need only `Get` and `Patch`, other methods are
-used in the same way - `Options` are exceptions.
+To control GPIOs on the RPI we need only `Get` and `Patch` methods, other 
+methods are used in the same way - `Options` is used for exceptions.
 
 I assume you have already installed Robot Framework. If response is "NO",
 I would like invite you to read the installation paragraph
```

Comment:
would like to

Reply:
I think we can omit `installation paragraph` since the link is called [Installation parapraph] (no words duplication)